csrc/scheduler/utils.h

```diff
@@ -832,5 +832,10 @@ TensorView* getUpCastInputOf(const TensorView* buffer_tv);
 //! See device_lower/analysis/tensor_producer_aliases.h
 TensorView* scheduleInputToSkipIntermediates(TensorView* tv);
 
+//! Utility enum to signify which direction
+//! transform propagation passes will propagate the transforms.
+//! For example, in sharding propagation or
+//! BoundedDirectionalTransformPropagator.
+enum class PropagateDirection { Backward = 0, Forward };
```

Comment:
```suggestion
enum class PropagateDirection { kBackward = 0, kForward };
```
https://google.github.io/styleguide/cppguide.html#Enumerator_Names